torchtitan/experiments/simple_fsdp/deepseek_v3/parallelize.py

```diff
@@ -19,10 +19,35 @@
 )
 from torchtitan.tools.logging import logger
 
-from ..backend import get_compile_backend
+from ..backend import get_compile_backend_and_passes
 
 from ..simple_fsdp import data_parallel, MixedPrecisionPolicy
 
+
+def get_fsdp_buckets(model) -> list[list[str] | str]:
```

Comment:
```suggestion
def get_transformer_block_buckets(model) -> list[list[str] | str]:
```
I omit "fsdp" for brevity and also because the buckets themselves don't have to be used for fsdp.